modules/ROOT/pages/api-groups-creating-groups.adoc

```diff
@@ -0,0 +1,80 @@
+= Create API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can bundle a set of APIs to enable ease-of-use and efficiency for your API consumers. To create a bundle, you first create an API Group and then add the relevant API instances to it. If you want your API Group to be discovered by consumers, you must publish it to Anypoint Exchange.
+
+You can create API Groups using the following steps:
+
+*Step 1:* <<create-api-groups,Create an API Group>>
+
+The first step to creating an API Group that can be used by consumers, is create an API Group. 
+
+*Step 2:* <<adding-api-instances-to-groups,Add API instances to the new API Group>>
+
+After you create the API Group, you then add API instances to this group for the first time.
+
+== Task Prerequisites
+
+Before you begin, you must have:
+
+* At least one API instance in your organization or business group.
+* API Group Administrator permissions for all the business groups and environments in which your API instances are located. 
++
+Otherwise, you may not find the required organization or business group in the list.
+
+[[create-api-groups]]
+== Create API Groups
+
+You can create an API Group for a specific organization and then add this group to multiple organizations, if required.
+
+To create API Groups:
+
+. From the left menu in API Manager, click *API Groups > Create API Groups*.
++
+The Create New Product window appears.
+. In the *API Group Name* field, specify a name for the API Group that you want to create: for example, social-media.
+. In the *API Group version* field, specify the version for the API Group, for example, `v1`.
+. Optionally, in the API Group instance label field, specify a label for the API Group, such as `Internal`, if it is to be used for internal purposes.
+. Click *Ok*.
++
+The *Creating a new group* page is displayed. You are now ready to <<adding-api-instances-to-groups,add API instances>> to the API Group.
+
+
+[[adding-api-instances-to-groups]]
+== Add API instances to API Groups
+
+Your API Group is not complete and functional unless you add the API instances to the group that you want to bundle. You can add API instances that belong in the same business group or organizations.
```

Comment:
Please add the environment type restriction

Reply:
Done! Thanks!